services/quests/docs/qdl_guide.md

```diff
@@ -310,21 +336,85 @@ To do this, you simply annotate the combat on win outcome. Below, you can see wh
 
 Note: In cases where adventurers won't heal at the end of combat, it's nice to warn them with an instruction node before combat starts so that they can make better choices - for example: `> You're on the run! You won't heal after this encounter.`
 
+### Decision cards
 
-### Roleplaying Cards
+Decisions allow you to add a more structure element of action to your quest. These can happen both in combat and during roleplay - in fact, they can happen automatically in combat if The Future expansion is enabled!
```

Comment:
What, not continuing the salsa metaphor? ;)

Reply:
I only included the salsa references to pico their gallos ^^